lib/pages/rendezvous/rendezvous_details_page.dart

```diff
@@ -0,0 +1,170 @@
+import 'dart:io' as io;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:matomo/matomo.dart';
+import 'package:pass_emploi_app/presentation/rendezvous/rendezvous_details_view_model.dart';
+import 'package:pass_emploi_app/redux/app_state.dart';
+import 'package:pass_emploi_app/ui/drawables.dart';
+import 'package:pass_emploi_app/ui/margins.dart';
+import 'package:pass_emploi_app/ui/strings.dart';
+import 'package:pass_emploi_app/ui/text_styles.dart';
+import 'package:pass_emploi_app/utils/platform.dart';
+import 'package:pass_emploi_app/widgets/buttons/secondary_button.dart';
+import 'package:pass_emploi_app/widgets/default_app_bar.dart';
+import 'package:pass_emploi_app/widgets/sepline.dart';
+import 'package:pass_emploi_app/widgets/text_with_clickable_links.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class RendezvousDetailsPage extends StatelessWidget {
+  final String rendezvousId;
+
+  RendezvousDetailsPage._(this.rendezvousId) : super();
+
+  static MaterialPageRoute<void> materialPageRoute(String rendezvousId) {
+    return MaterialPageRoute(builder: (context) => RendezvousDetailsPage._(rendezvousId));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final platform = io.Platform.isIOS ? Platform.IOS : Platform.ANDROID;
+    return StoreConnector<AppState, RendezvousDetailsViewModel>(
+      converter: (store) => RendezvousDetailsViewModel.create(store, rendezvousId, platform),
```

Comment:
Ce changement (avant on passait un rdv, maintenant on passe l'id et c'est ce component qui va récupérer le rdv) me fait beaucoup penser à [cette guideline de la doc redux.js.org](https://redux.js.org/style-guide/style-guide#connect-more-components-to-read-data-from-the-store) ➡️  Plutôt que d'avoir un component qui récupère plein de données, on a plein de components qui récupèrent un peu de données. 

Je me demande si on voudrait pousser ça plus loin ? Ca semble logique ici - un écran est responsable de sa donnée, mais est-ce qu'on voudrait aller aussi loin que ce que la doc recommande et faire ça par exemple au niveau des listes ?

Sur cette feature ça pourrait donner un truc du genre : 
Le `RendezvousListViewModel` a une list d'ids 
Dans la page, pour chaque item de la liste, on met un `StoreConnector` au niveau du `CalendarCard` , qui créé un `RendezvousCardViewModel` à partir de l'id. 

Ils parlent de gains de perf dans la doc, mais vu qu'on splitte les responsabilités je vois surtout un gain au niveau de la lisibilité des view models et de leur test : on aura d'un coté la gestion du contenu de la page, de l'autre le contenu d'un item. 

Chaud d'en discuter en tout cas 😄